crates/pkgcraft-tools/src/subcmds/pkg/source.rs

```diff
@@ -372,7 +372,10 @@ fn cumulative(limit: u32, targets: PkgTargets, cmd: &Command) -> anyhow::Result<
     let mut run = 0;
     let mut values = vec![];
     let mut stdout = io::stdout().lock();
-
+    rayon::ThreadPoolBuilder::new()
```

Comment:
This can be handled via pkgcraft::utils::bounded_thread_pool, look at `pk pkg metdata` or `pk repo metadata regen` for example usage.

Reply:
Sure